### PR TITLE
Use get-tsconfig for tsconfig parsing

### LINKED ISSUE #72: Tsconfig parser does not support Node.js-style resolution of the `extends` field
#65 removed support for Node.js-style resolution of the tsconfig `extends` field – something that [TypeScript can do](https://www.typescriptlang.org/tsconfig#extends). Implementing this from-scratch is not entirely trivial, but [get-tsconfig](https://github.com/privatenumber/get-tsconfig) has it all. @davestewart Since you decided against `get-tsconfig`, I'm not sure if there's a reason not to use it in this project? If not, I'm happy to submit a PR (I'm desperately depending on module resolution in `extends` :smiling_face_with_tear:). Just drop me a thumbs-up reaction here if `get-tsconfig` would also work.

### PR DESCRIPTION
**Pull request by bjoluc**
Closes #72 by utilizing the [get-tsconfig](https://github.com/privatenumber/get-tsconfig) package to parse ts/jsconfig files.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -142,31 +143,12 @@ function loadSettings () {
...
-        return true
-      }
-    }
-
-    // if no paths, check extends
-    if (json.extends) {
-      return loadConfig(resolve(Path.dirname(path), json.extends))
-    }
-  }
+  const { compilerOptions: { baseUrl = '', paths = {} } } = parseTsconfig(path)
```
Comment (bjoluc) [inline src/index.js:146]: If you wonder why I removed the `if (compilerOptions)` check: `parseTsconfig` either throws an exception if `path` does not exist, or returns an object containing `compilerOptions`. If the config file contains syntax errors that prevent `get-tsconfig` from parsing it, default `compilerOptions` are returned. Seems to be the way TS handles this too.

Comment (davestewart) [reply src/index.js:146]: Do we need to catch and represent the error as an Alias HQ issue if path does not exist, or will it be obvious from the context what it is and what needs to be done by the user?

Comment (davestewart) [reply src/index.js:146]: Cool!

I'll treat this locally later then it looks good to go

**Comment by bjoluc** [comment]
I hope you had a great vacation @davestewart! Can I do anything to help with this?

**Comment by davestewart** [comment]
Let me get on this!

**Comment by bjoluc** [comment]
Sorry to bump again :smiling_face_with_tear:

**Comment by davestewart** [comment]
Don't be!!! 😅

**Comment by bjoluc** [comment]
:wave: :innocent:

**Comment by davestewart** [comment]
The good news is I released my project!

So... I PROMISE to do this first thing tomorrow 🙌

**Comment by davestewart** [comment]
It's done! Releasing now. Thanks for being so patient and good-natured 😄

**Comment by bjoluc** [comment]
Awesome, thanks! :heart:

**Pull request merged**